archive/options.go

```diff
@@ -16,7 +16,7 @@ package archive
 
 import (
 	"github.com/lirm/aeron-go/aeron/idlestrategy"
-	logging "github.com/op/go-logging"
+	"go.uber.org/zap/zapcore"
```

Comment:
Why not use the same abstraction here? Seems to be the only exception.

Reply:
I could remove the archive options that hold the loglevel as they're currently unused. Let me know if you would prefer that for now.

It's my intention to provide methods from the archive to allow runtime control over the logging not only of the level (as foreshadowed) but also in the future to provide access to some further options that make use of the available zap runtime features.

This is for now a minimalist set of code changes across aeron-go that switch to zap. I imagined over time that you may want to adopt more of it generally and more directly but this should get over the immediate concurrency issues without large code changes.

Reply:
I guess it's ok as a step in migration process and it is isolated.